modules/virt-importing-vm-cli.adoc

```diff
@@ -98,37 +98,40 @@ spec:
   source:
     ovirt:
       vm:
-        id: <80554327-0569-496b-bdeb-fcbbf52b827b> <3>
-      mappings: <4>
+        id: <source-vm-id> <3>
+        name: <source-vm-name> <4>
+      cluster:
+        name: <source-cluster-name> <4>
+      mappings: <5>
         networkMappings:
           - source:
-              name: <rhv-logical-network>/<vnic-profile> <5>
+              name: <source-logical-network>/<vnic-profile> <6>
             target:
-              name: <target-network> <6>
+              name: <target-network> <7>
             type: pod
-        storageMappings: <7>
+        storageMappings: <8>
           - source:
-              name: <rhv-storage-domain> <8>
+              name: <source-storage-domain> <9>
             target:
-              name: <target-storage-class> <9>
+              name: <target-storage-class> <10>
         diskMappings:
           - source:
-              id: <8181ecc1-5db8-4193-9c92-3ddab3be7b05> <10>
+              id: <source-vm-disk-id> <11>
             target:
-              name: <target-storage-class> <11>
+              name: <target-storage-class> <10>
 EOF
 ----
 <1> If you create a ResourceMapping CR, uncomment the `resourceMapping` section.
 <2> The `targetVmName` value is limited to 63 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
```

Comment:
I'll raise this on the other PR as well but the updated release note says 23 instead of 63.
I see both are used in the bug.

Reply:
Forgot to update the release note. It's supposed to be 63